src/forms.css

```diff
@@ -328,6 +328,7 @@ because it seems to be more cross-browser consistent with all HTML5 input types
   margin: 0;
   border: 0;
   background: var(--transparent);
+  cursor: pointer;
```

Comment:
👍 - this does make me wonder if the handle should have a _grabber_ cursor that transitions into grabbing on active?